network/topics/scoring.go

```diff
@@ -249,3 +253,13 @@ func filterCommitteesForTopic(topic string, committees []*storage.Committee) []*
 	}
 	return topicCommittees
 }
+
+// truncateStats returns the subnets in a small format topicNum=ti,fmd,mmd,imd
+func truncateStats(filtered map[string]*pubsub.TopicScoreSnapshot) []string {
```

Comment:
just to be a bit more specific (i know long names are not ideal but i think they're very helpful if they're only used once)
```suggestion
func formatInvalidMessageStats(filtered map[string]*pubsub.TopicScoreSnapshot) []string {
```